src/oidcop/oauth2/token.py

```diff
@@ -477,7 +477,9 @@ def process_request(self, request: Optional[Union[Message, dict]] = None, **kwar
             name=_context.cookie_handler.name["session"],
             sub=_session_info["grant"].sub,
             sid=_context.session_manager.session_key(
-                _session_info["user_id"], _session_info["user_id"], _session_info["grant"].id,
+                _session_info["user_id"],
+                _session_info["user_id"],
```

Comment:
Shouldn't this be `_session_info["client_id"]`?

Reply:
yes

Reply:
I'm going to fix this, thank you @nsklikas

Reply:
mmhh .. it's this way also in the current master branch!
https://github.com/IdentityPython/oidc-op/blob/05611f16a31ce9092b83b00888cebc61cf98ca55/src/oidcop/oauth2/token.py#L480

ok, consider it fixed